tools/release/bump-version-and-create-release-note.sh

```diff
@@ -35,6 +35,9 @@ docker run --rm -it -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-chan
 # bump the version
 echo "${RELEASE_VERSION}" >VERSION
 
+#update aws-otel-collector msi file
+sed -i '' "s/\(<Product [^>]*Version=\"\)[^\"]*\"/\1${RELEASE_VERSION:1}\"/" tools/packaging/windows/aws-otel-collector.wxs
```

Comment:
Can you post test results for this oneliner?

Reply:
I tried running it locally but the file was not updated
```
RELEASE_VERSION=0.26.0 sed -i '' "s/\(<Product [^>]*Version=\"\)[^\"]*\"/\1${RELEASE_VERSION:1}\"/" tools/packaging/windows/aws-otel-collector.wxs
```

Reply:
Also, does `RELEASE_VERSION` usually include a `v` prefix? Do we need to remove that before doing the search and replace?

Reply:
Also, please add a comment saying that this only supports `[0-9]*.[0-9]*.[0-9]*` version pattern. If we decide to release with a full sem ver `vX.XX.XX-prerelease` then the `.wxs` file will need to be manually updated.

Reply:
> I tried running it locally but the file was not updated

I have updated the sed pattern. Here are the results

### .wxs file before
<img width="578" alt="Screen Shot 2023-02-01 at 4 51 58 PM" src="https://user-images.githubusercontent.com/28486681/216204791-dfa20e58-bc3f-43a6-a16f-a198150d5612.png">

### .wxs file after (as expected)
<img width="936" alt="Screen Shot 2023-02-01 at 4 53 16 PM" src="https://user-images.githubusercontent.com/28486681/216205012-8bac9151-c689-4a3c-bf6b-34ae45dc7153.png">

Reply:
> Also, does RELEASE_VERSION usually include a v prefix? Do we need to remove that before doing the search and replace

the RELEASE_VERSION has 'v' prefix it is but this first character is removed by `${RELEASE_VERSION:1}`

> Also, please add a comment saying that this only supports [0-9]*.[0-9]*.[0-9]* version pattern. If we decide to release with a full sem ver vX.XX.XX-prerelease then the .wxs file will need to be manually updated.

Added it in PR description along with comment inside the shell script

Reply:
`RELEASE_VERSION=v0.26.0 sed -i '' "s/^Version\=\"[0-9]*.[0-9]*.[0-9]*\"/Version=\"${RELEASE_VERSION:1}\"/" tools/packaging/windows/aws-otel-collector.wxs`
Results in `Version=".26.0"`  when I run it locally.

Reply:
Disregard above. This was a local environment issue. I can confirm it works.